docs/release-notes/8.12.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[release-notes-header-8.12.0]]
+== 8.12
+
+[discrete]
+[[release-notes-8.12.0]]
+=== 8.12.0
+
+[discrete]
+[[known-issue-8.12.0]]
+==== Known issues
+
+There are no known issues in 8.12.0.
```

Comment:
```suggestion

// tag::known-issue-4582[]
[discrete]
.[Title]
[%collapsible]
====
*Details* +
On January 9, 2023, a known issue was discovered with the Intelligence page. When users filter indicator data from the different places on the page, the KQL bar shows the correct field name, but the field value is "Warning". 
*Workaround* +
There is no workaround for this bug.
====
// end::known-issue-[PR number][]
```

Reply:
Sorry to jump in a bit late here. I think we can add https://github.com/elastic/kibana/issues/173958 as a known issue for 8.12. 

**Details**

When editing a rule using an index pattern and attempting to edit the query filter, a  data view selector is displayed and the filter does not populate. 

**Workaround**

Select to edit the query filter using DSL.

cc @e40pud (working on a fix) @peluja1012

Reply:
@yctercero and @e40pud - is there any way we can simplify the known issue summary so it's more straightforward? I took a look at https://github.com/elastic/kibana/issues/173958 and am not all that clear on whether the issue is a UI bug, or if the data view filter function doesn't work at all. 

If it's a UI bug, would something like the following work:
When updating the Alerts page KQL query bar filter, you might encounter a UI bug when selecting the default Security data view option.  

Then for the workaround, do users need to click the **Edit as Query DSL** option in the upper right corner of the filter modal? 
<img width="1124" alt="Screenshot 2024-01-12 at 2 49 45 PM" src="https://github.com/elastic/security-docs/assets/79124755/03342d34-0028-4b15-b585-3dfc7d42aef1">

Also, if this bug only prevents users from selecting the default Security data view filter, should we offer a DSL that offers the same filtering functionality? If we don't want to get that specific, can we offer them any general guidance so they have a way to proceed with filtering?